ntp-udp/src/raw_socket.rs

```diff
@@ -253,14 +251,14 @@ mod recv_message {
         };
 
         if mhdr.msg_flags & libc::MSG_TRUNC > 0 {
-            warn!(
+            tracing::info!(
                 max_len = packet_buf.len(),
                 "truncated packet because it was larger than expected",
             );
         }
 
         if mhdr.msg_flags & libc::MSG_CTRUNC > 0 {
-            warn!("truncated control messages");
+            tracing::info!("truncated control messages");
```

Comment:
I think this should only happen because of bugs on our end. As such, I think it should be higher than info, at least warn, and we should probably say something about this likely being a bug.

Reply:
Choice: warn